assets/sass/components/dashboard/wizard/_googlesitekit-wizard-step.scss

```diff
@@ -92,10 +92,6 @@
 	}
 
 	.googlesitekit-wizard-step__back {
-		margin-left: $grid-gap-phone;
-
-		@media (min-width: $bp-desktop) {
-			margin-left: $grid-gap-desktop;
-		}
+		margin-left: $grid-gap-phone / 2;
```

Comment:
Not sure what's happening here. Can you help me understand this change?

Reply:
Sure.

This style is for the `Back` button in the legacy setup, here:

![image](https://github.com/google/site-kit-wp/assets/20284937/7a5337ab-4932-496e-8ac4-161880055928)

While using the link, it has a `margin-left` value of `24px` (`$grid-gap-desktop`).

However, when we go on to replace the link with our tertiary button (which already has a horizontal padding of `16px`), using the same `margin-left` value of `24px` creates a much larger white space between the actions. To prevent this, the above change aims to preserve the same `24px` spacing between the actions by applying a `margin-left` value of `8px` (`$grid-gap-phone / 2`), effectively preserving the same `24px` spacing (`16px` padding from button + `8px` `margin-left`). This is what we get:

![image](https://github.com/google/site-kit-wp/assets/20284937/baff3391-8db5-45ee-a169-744a16c1ab9c)

In mobile, `develop` applies a margin-left of `16px`. However, since our tertiary button already has a horizontal padding of `16px`, to create some space, we apply the same `8px` margin-left in all viewports.

Please let me know if this makes sense, thank you!

Reply:
Thank you for the explanation. However, since this is effecting all viewports and `8px` is more or less a literal value here, I think we should use that instead of `$grid-gap-phone / 2` which is hard to reason about in this context. What do you think?

Reply:
Thank you for the explanation. However, since this is effecting all viewports and `8px` is more or less a literal value here, I think we should use that instead of `$grid-gap-phone / 2` which is hard to reason about in this context. What do you think?

Reply:
Updated, please take a look, thank you!